ViewModel/LibraryViewModel.swift

```diff
@@ -171,14 +173,23 @@ final class LibraryViewModel: ObservableObject {
 
     private func fetchData() async throws -> Data? {
         do {
-            let request = URLRequest(url: Self.catalogURL, timeoutInterval: 20)
+            var request = URLRequest(url: Self.catalogURL, timeoutInterval: 20)
+            request.allHTTPHeaderFields = ["If-None-Match": Defaults[.libraryETag]]
             let (data, response) = try await self.urlSession.data(for: request)
             guard let response = response as? HTTPURLResponse else { return nil }
-            guard response.statusCode == 200 else {
+            switch response.statusCode {
+            case 200..<300:
```

Comment:
Use `200`. There's no reason to expect other and this could lead to issue

Reply:
Ok, see it updated.